common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/actions/models/CFAction.kt

```diff
@@ -0,0 +1,13 @@
+package io.xorum.codeforceswatcher.features.actions.models
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CFAction(
+        val timeSeconds: Long,
+        val blogEntry: BlogEntry,
+        val comment: Comment? = null
+) {
+    val id
```

Comment:
Actually, all your fields are `val`, so can't be changed. This means you can safely write `val id = hashCode()`.